app/models/searchApi/OwnerAuthorisation.scala

```diff
@@ -20,14 +20,12 @@ import play.api.libs.json.Json
 
 case class OwnerAuthorisation(
       authorisationId: Long,
-      status: String,
       submissionId: String,
       uarn: Long,
       address: String,
       localAuthorityRef: String,
-      agents: Seq[OwnerAuthAgent]
-) {
-
+      agents: Seq[OwnerAuthAgent]) {
+  val status = "APPROVED"
```

Comment:
dont know about this. I think we are keeping authorisation status. its only agent representation status that is going.